packages/customers/src/Nav.tsx

```diff
@@ -30,22 +30,22 @@ const Nav: FC = () => {
   const t = useTranslation();
   return (
     <>
-      <AppNavLink
+      <NavLink
         end={!isActive}
         to={AppRoute.Customers}
         icon={<Customers fontSize="small" />}
         text={t('app.customers')}
       />
       <Collapse in={isActive}>
         <List>
-          <AppNavLink
+          <NavLink
             end={true}
             to={RouteBuilder.create(AppRoute.Customers)
               .addPart(AppRoute.CustomerInvoice)
               .build()}
             text={t('app.customer-invoice')}
           />
-          <AppNavLink
+          <NavLink
```

Comment:
Changed from `AppNavLink` -> `NavLink` because there wasn't enough offroad diffing in this PR